Added NBFT table support

johnmeneghini opened the pull request:
Added support for parsing and printing the contents of the NBFT table (per NVMe-oF boot specification v1.0).

Signed-off-by: Stuart Hayes <stuart_hayes@dell.com>

mwilck commented:
Why is the target branch not simply `master` ?

igaw commented:
This looks in reasonable shape to send to upstream for review. I only quickly glanced over the code and spotted a few things I am certainly going to comment on with my upstream hat on when I see the official PR against the upstream project. Though I don't see a reason not to send it upstream at this point and get it exposed to more eyes to review

johnmeneghini commented:
Closing this merge request.  See the upstream merge request: 

https://github.com/linux-nvme/libnvme/pull/572

Note: I've also updated the timberland SIG repository and rebased this branch onto v1.3.
